### PR TITLE
[FIX] Collaborative: misc fixes

### PR DESCRIPTION
**Pull request by pro-odoo**
## Description:

description of this task, what is implemented and why it is implemented that way.

Odoo task ID : [TASK_ID](https://www.odoo.com/web#id=TASK_ID&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_lt("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

**Comment by rrahir** [comment]
@robodoo r+

**Comment by LucasLefevre** [comment]
robodoo r-

**Comment by LucasLefevre** [comment]
just running a bit more the monkey party before merging.
Already found other issues. Let me check they aren't related

**Comment by LucasLefevre** [comment]
@rrahir PR 16.0 ready. With commits from 15.2 cherry-picked and some tests added in one commit

**Comment by rrahir** [comment]
test caput :'(

**Comment by rrahir** [comment]
COuld you also add a commit to ignore the reaction to 'CLIENT_MOVED' when we sent it in the first place ?

File: src/collaborative/session.ts [hunk 1]
```diff
@@ -383,6 +382,10 @@ export class Session extends EventBus<CollaborativeEvent> {
         this.sendPendingMessage();
         break;
     }
+    if (message.type === "CLIENT_MOVED" && message.client.id === this.clientId) {
```
[the remark this answers is not in the document]
Comment (LucasLefevre) [reply src/collaborative/session.ts]: updated :)

**Comment by rrahir** [comment]
@robodoo r+

**Comment by pro-odoo** [comment]
robodoo rebase-ff

**Comment by rrahir** [comment]
@robodoo rebase-ff

**Pull request closed without merging**